Kickstarter-iOS/Views/Controllers/PledgeSummaryViewController.swift

```diff
@@ -69,10 +71,10 @@ final class PledgeSummaryViewController: UIViewController {
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToEdgesInParent()
 
-    _ = ([self.titleLabel, self.adaptableStackView], self.rootStackView)
+    _ = ([self.adaptableStackView, self.confirmationLabel], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = ([self.termsTextView, self.totalStackView], self.adaptableStackView)
+    _ = ([self.titleLabel, self.totalStackView], self.adaptableStackView)
```

Comment:
Should we rename this title label to total? It was kind of confusing knowing what it was.

Reply:
I think in this case it refers to the title of the section. I think it's better to name a UI element according to what it is rather than what it says because the latter can change.